scala3doc/test/dotty/dokka/SignatureTest.scala

```diff
@@ -98,7 +98,7 @@ abstract class SignatureTest(
           case unexpectedRegex(signature) => findName(signature, kinds).map(Unexpected(_))
           case expectedRegex(signature) => findName(signature, kinds).map(Expected(_, signature))
           case signature =>
-            findName(signature, kinds).map(Expected(_, commentRegex.replaceAllIn(signature, "").compactWhitespaces))
+            findName(signature, kinds).map(Expected(_, commentRegex.replaceAllIn(signature, "").compactWhitespaces.reverse.dropWhile(List('{', ':').contains(_)).reverse))
```

Comment:
Can we break this into at least 2 lines?

Reply:
Ok, will change